src/models/tag.ts

```diff
@@ -300,7 +298,7 @@ const validateTag = (tag: any): true => {
 
   if (err) {
     Sentry.captureException(err);
-    throw new ApolloError(err);
+    throw new Error(err);
```

Comment:
I think this might be getting captured twice with the sentry plugin. I think we might also want this to be a `GraphQLError` to match the old behavior. Plain JS errors get masked to a InternalServerError.

Reply:
Seeing some similar threads here, will put it on my list to double check the behavior that's intended.